### PR TITLE
Fix: #10870

### PR DESCRIPTION
**Pull request by prathamranjansingh**
Fixed the isuue- [docs]: Wrong roadmap link in README #10870

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +69,7 @@ docker run \
   tooljet/try:EE-LTS-latest
 ```
 
-*For users upgrading their ToolJet version, we recommend choosing the LTS version over the latest version. The LTS version ensures stability with production bug fixes, security patches, and performance enhancements.*
+_For users upgrading their ToolJet version, we recommend choosing the LTS version over the latest version. The LTS version ensures stability with production bug fixes, security patches, and performance enhancements._
```
Comment (RudraPratik30) [inline README.md:72]: Hey @prathamranjansingh!
Can you please revert this change? Use ** instead of _ _ for this line.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @prathamranjansingh, please make the mentioned change. Thanks!

**Review by RudraPratik30 — APPROVED** [review]
Hey @prathamranjansingh!
Approving changes from my end and forwarding this to the team for final review.
Thank you for contributing, your PR will be merged by the end of this week.

**Comment by prathamranjansingh** [comment]
I am really sorry sir that you had to do the changes. Next time i will be
careful and ask someone to assign me the issue also. Thank you sir

File: package.json
Comment (RudraPratik30) [inline package.json:1]: Can you check what's the change in this file and revert it?

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]

**Comment by RudraPratik30** [comment]
Also @prathamranjansingh can you please let me know what changes you have made after the PR was approved?

**Review by RudraPratik30 — APPROVED** [review]
Hey @prathamranjansingh, approving the PR from my end and forwarding it to the team for final review. Your PR will be merged by the end of this week.
You are not required to push any more commits once your PR is approved, this will revoke the stable review.
Thanks for contributing!

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me.

**Pull request merged**